.factory/automation.yml

```diff
@@ -79,6 +79,15 @@ build:
         export DEPLOY_PIP_PASSWORD=$REPO_VATICLE_PASSWORD
         bazel run --define version=$(git rev-parse HEAD) //grpc/python:deploy-pip -- snapshot
       dependencies: [build, build-dependency]
+    deploy-crate-snapshot:
+      filter:
+        owner: vaticle
+        branch: master
+      image: vaticle-ubuntu-22.04
+      command: |
+        export DEPLOY_CRATE_TOKEN=$REPO_VATICLE_CRATES_TOKEN
```

Comment:
`CRATE` = the package type (similar to `pip`, `npm`)
`CRATES` = the registry type (similar to `PyPi`, `npm`)